trulens_eval/trulens_eval/feedback/groundtruth.py

```diff
@@ -168,15 +172,17 @@ def agreement_measure(
                 prompt, response, ground_truth_response
             )
             ret = (
-                re_0_10_rating(agreement_txt) / 10,
+                re_configured_rating(agreement_txt) / 3,
```

Comment:
I'm guessing re_configured_rating is missing `max_value=3` here

Reply:
`re_configured_rating` is default to likert_0_3 now.


But yes this is in incomplete shape and I've added near-term action items in the PR description including mapping output_space to score range